src/main/java/lotto/model/enums/WinningMoney.java

```diff
@@ -0,0 +1,38 @@
+package lotto.model.enums;
+
+import java.util.Arrays;
+
+public enum WinningMoney {
+    NON_MATCH(0, false, 0),
+    THREE_MATCH(3, false, 5000),
+    FOUR_MATCH(4, false, 50000),
+    FIVE_MATCH(5, false, 1500000),
+    FIVE_MATCH_BONUS(5, true, 30000000),
+    SIX_MATCH(6, false, 2000000000);
```

Comment:
이와 같이 숫자의 자리수가 많아질 경우 `_`로 구분하면 훨씬 가독성이 좋아집니다!
```suggestion
    SIX_MATCH(6, false, 2_000_000_000);
```

Reply:
하나 배워갑니다... 모니터에 손가락 대고 하나씩 세던 제 모습을 더이상 안 봐도 되겠네요 ㅎㅎ

Reply:
이 부분에 대해서 생각하지 못했는데, 감사합니다..!